src/summaries/summaries-nav.component.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect } from "react";
+import PatientChartOverview from "../summaries/overview/patient-chart-overview.component";
+import { levelTwoRoutes } from "../chart-review/level-two-routes.component";
+import { Breadcrumbs } from "../breadcrumbs/breadcrumbs.component";
+import { Route, Link, Redirect, useHistory, useParams } from "react-router-dom";
+import styles from "./summaries-nav.css";
+import { RouteComponentProps } from "react-router";
+
+export default function SummariesNav(props: any) {
+  let { patientUuid } = useParams();
+  const history = useHistory();
+
+  const navItems = [
+    {
+      name: "Overview",
+      path: `/patient/${patientUuid}/chart/summaries/overview`
+    },
+
+    {
+      name: "HIV",
+      path: `/patient/${patientUuid}/chart/summaries/hiv`
+    }
+  ];
+
+  let i = 0;
+  navItems.forEach((item, index) => {
+    if (item.path === history.location.pathname) {
+      i = index;
+    }
+  });
```

Comment:
Looks like this code is not being used.

Reply:
It gets used on the initial render of the component. 
There are three scenarios I've been trying to find ways (somewhat unsuccessfully) to account for:

1. The user navigates from outside patient chart to /chart. In this case, there should be no tabs selected. 
2. The user directly navigates to a url like : /chart/summaries/overview, in which case the summaries and overview tabs should be selected (this would be handled by the above code) 
3. The user is been navigating within a chart. For example, a user clicks on summaries --> overview, then clicks on the (main nav).orders and then clicks back to (main nav).summaries. In this case, I want the last summary tab that the user was on to be displayed. (this is the trickiest scenario).